manifests-maistra/1.0.0/maistraoperator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -69,15 +69,15 @@ metadata:
                 "traceSampling": 100.0
               },
               "kiali": {
-                "enabled": false
+                "enabled": true
               },
               "grafana": {
-                "enabled": false
+                "enabled": true
               },
               "tracing": {
                 "enabled": true,
                 "jaeger": {
-                  "template": "all-in-one"
+                  "template": "production-elasticsearch"
```

Comment:
Is this the appropriate value for Maistra? Especially without an elastic search version?

Reply:
This would match the profile defaults since it is set in base.  It is probably better to have the all-in-one profile for maistra and production-elasticsearch for servicemesh but this would also need a change to the profiles.

Reply:
I'll leave my review as requesting changes until we get an answer to this one

Reply:
jaeger.template in the maistra csv is now being set to "all-in-one"